apps/samradsgatt/proxy.config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "/app/samradsgatt/api/graphql": {
```

Comment:
Should copy this file from the web project. It basically maps "/api" to the graphql api url.

That way you can call the API locally like in production where this project is on /samradsgatt and the api is on the same domain /api.